internal/validations/clusterlogging/validate_clusterlogging_spec.go

```diff
@@ -15,8 +21,16 @@ func validateClusterLoggingSpec(cl v1.ClusterLogging) error {
 		return nil
 	}
 	spec := cl.Spec
+
 	if spec.Forwarder != nil || spec.LogStore != nil || spec.Curation != nil || spec.Visualization != nil {
-		return errors.NewValidationError("Only spec.collection is allowed when using multiple instances of ClusterLogForwarder: %s/%s", cl.Namespace, cl.Name)
+		key := types.NamespacedName{Name: cl.Name, Namespace: cl.Namespace}
+		clf := runtime.NewClusterLogForwarder(cl.Namespace, cl.Name)
```

Comment:
I don't think this needs to be anything other then "&logging.ClusterLogForwarder{}" since the "get" will fill in all the pieces if it is found using the key